src/lwc/geFormField/geFormField.js

```diff
@@ -202,6 +204,9 @@ export default class GeFormField extends LightningElement {
             // the RecordType Name
             fieldAndValue[this.formElementName] =
                 this.objectDescribeInfo.recordTypeInfos[this.value].name;
+        } else if (this.isPicklist){
+            // If the displayed value of the picklist is '--None--' treat the value as blank.
+            fieldAndValue[this.formElementName] = (this.value === this.CUSTOM_LABELS.commonLabelNone) ? '' : this.value;
```

Comment:
Minor. Could we break after the equals on this line?

Reply:
done!